exporters/etw/test/etw_tracer_test.cc

```diff
@@ -459,6 +468,26 @@ TEST(ETWTracer, AlwayOffSampler)
   EXPECT_EQ(span->GetContext().IsSampled(), false);
 }
 
+TEST(ETWTracer, AlwayOffTailSampler)
+{
+  std::string providerName = kGlobalProviderName; // supply unique instrumentation name here
+  std::unique_ptr<sdk::trace::Sampler> always_on{new sdk::trace::AlwaysOnSampler()};
+  sdk::trace::IdGenerator *id_generator = new MockIdGenerator();
+  std::unique_ptr<TailSampler> always_off_tail{new AlwaysOffTailSampler()};
+  exporter::etw::TracerProvider tp
+    ({
+      {"enableTraceId", true},
+      {"enableSpanId", true},
+      {"enableActivityId", true},
+      {"enableRelatedActivityId", true},
+      {"enableAutoParent", true}
+     },
+     std::move(always_on),
```

Comment:
Could we start a span and test it is indeed dropped?

Reply:
Yes I was thinking to mock the actual ETW transport layer and do the test. In current design, this is not easy to do, perhaps can be taken separately if it is fine?